test/seed.js

```diff
@@ -235,22 +235,40 @@ contract('Seed', (accounts) => {
                         isWhitelisted,
                         fee
                     );
-                    await fundingToken.transfer(buyer2, smallBuyAmount, {from:setup.root});
-                    await fundingToken.approve(setup.data.seed.address, smallBuyAmount, {from:buyer2});
+                    // Also changed the below staatement, so that on buy() the limit does not exceeds
+                    await seedToken.transfer(setup.data.seed.address, (new BN(cap,10)).div(new BN(price,10)).mul(new BN(pct_base,10)), {from:setup.root});
+                    await fundingToken.approve(setup.data.seed.address, (new BN(smallBuyAmount,10)).mul(new BN(price,10)).div(new BN(pct_base,10)), {from:buyer2});
+                    // the buyer buys seed token here, to test retrieveFundingTokens()
+                    await setup.data.seed.buy(toWei('900'), {from:buyer2});
                 });
                 it('can only be called by admin', async () => {
                     await expectRevert(
                         setup.data.seed.close(),
                         "Seed: caller should be admin"
                     );
                 });
-                it('transfers all tokens to the admin', async () => {
-                    let ftBalance = await fundingToken.balanceOf(setup.data.seed.address);
+                it('transfers seed tokens to the admin', async () => {
                     let stBalance = await seedToken.balanceOf(setup.data.seed.address);
                     await setup.data.seed.close({from:admin});
-                    expect((await fundingToken.balanceOf(admin)).toString()).to.equal(ftBalance.toString());
                     expect((await seedToken.balanceOf(admin)).toString()).to.equal(stBalance.toString());
                 });
+                // Added two test.
+                // Checks if the funding tokens are still in the contract
+                it('donot transfer funding tokens to the admin', async () => {
+                    let ftBalance = await fundingToken.balanceOf(setup.data.seed.address);
+                    expect((await fundingToken.balanceOf(setup.data.seed.address)).toString()).to.equal(ftBalance.toString());
+                });
+                // Checks if the buyer can retrieve token back even the contract is closed
+                it('returns funding tokens to buyer', async () => {
+                    expect((await fundingToken.balanceOf(buyer2)).toString()).to.equal('0');
+
+                    let tx = await setup.data.seed.retrieveFundingTokens({from:buyer2});
+                    setup.data.tx = tx;
+
+                    expectEvent.inTransaction(setup.data.tx.tx, setup.data.seed, 'FundingReclaimed');
+                    expect((await fundingToken.balanceOf(buyer2)).toString()).to
+                        .equal(((new BN(smallBuyAmount,10)).mul(new BN(price,10)).div(new BN(pct_base,10))).toString());
```

Comment:
pls avoid using magic values in tests

Reply:
I referred for this from a test above "returns funding tokens to buyer" at seed.js:202.